test_projects/documented/test/example_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule UnifiedTest do
+  use ExUnit.Case
+
+  alias ElixirSense.Core.Introspection
+
+  setup do
+    %{}
+  end
+
+  test "nif functions are documented" do
+    check_doc(:begin_file_documented_function)
+    check_doc(:inside_file_documented_function)
+    check_doc(:invalid_double_documented_function)
+    check_doc_false(:inside_file_undocumented_function)
+    check_doc(:after_undocumented_documented_function)
+    check_doc_false(:undocumented_false_function)
+    check_doc(:end_file_documented_function)
+  end
+
+  defp check_doc(fun_atom) do
+    assert get_doc(Example, fun_atom) == "Test #{fun_atom} documentation"
+  end
+
+  defp check_doc_false(fun_atom) do
+    assert get_doc(Example, fun_atom) == "false"
```

Comment:
Writing `@doc "false"` should not have this same effect, as writing `@doc false`, but this line suggests, that these 2 works this same way. Have you looked at generated documentation, to see, how it looks like there?